backend/eventloop/shared_resource_loop/sharedresourceloop_managedenv_test.go

```diff
@@ -438,7 +439,7 @@ var _ = Describe("SharedResourceEventLoop Test", func() {
 			defer mockCtrl.Finish()
 			mockClient := mocks.NewMockClient(mockCtrl)
 
-			mockClient.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any()).Return(fmt.Errorf("fake unable to connect"))
+			mockClient.EXPECT().List(gomock.Any(), gomock.Any()).Return(fmt.Errorf("fake unable to connect"))
```

Comment:
Get -> List, as we are now listing namespaces, rather than getting a serviceaccount.